os_dep/linux/ioctl_cfg80211.c

```diff
@@ -3212,7 +3212,8 @@ static int cfg80211_rtw_scan(struct wiphy *wiphy
 	_adapter *padapter;
 	struct wireless_dev *wdev;
 	struct rtw_wdev_priv *pwdev_priv;
-	struct mlme_priv *pmlmepriv = NULL;
+	struct mlme_priv *pmlmepriv = NULL, *buddy_mlmepriv;
```

Comment:
```suggestion
	struct mlme_priv *pmlmepriv = NULL;
	struct mlme_priv *buddy_mlmepriv;
```

Так будет нагляднее при сравнении версий

Reply:
fixed